src/v1/controllers/EnrollWebauthnController.js

```diff
@@ -85,11 +85,13 @@ export default FormController.extend({
             excludeCredentials: getExcludeCredentials(activation.excludeCredentials),
           });
 
-          self.webauthnAbortController = new AbortController();
+          if (typeof AbortController !== 'undefined') {
```

Comment:
if we require all instances of `AbortController` to have an existence check, could we add a linter rule (probably would be custom) to enforce this?

Reply:
this runtime existence check let the code passes the `compat/compat` lint check